include/rekey/rnp_key_store.h

```diff
@@ -208,14 +209,18 @@ pgp_key_t *rnp_key_store_import_signature(rnp_key_store_t *        keyring,
 
 bool rnp_key_store_remove_key(rnp_key_store_t *, const pgp_key_t *, bool);
 
-pgp_key_t *rnp_key_store_get_key_by_id(rnp_key_store_t *, const unsigned char *, pgp_key_t *);
+pgp_key_t *rnp_key_store_get_key_by_id(rnp_key_store_t *   keyring,
+                                       const pgp_key_id_t &keyid,
+                                       pgp_key_t *         key);
 
 bool rnp_key_store_get_key_grip(const pgp_key_material_t *, pgp_key_grip_t &grip);
 
 const pgp_key_t *rnp_key_store_get_key_by_grip(const rnp_key_store_t *,
                                                const pgp_key_grip_t &);
 pgp_key_t *      rnp_key_store_get_key_by_grip(rnp_key_store_t *, const pgp_key_grip_t &);
-pgp_key_t *      rnp_key_store_get_key_by_fpr(rnp_key_store_t *, const pgp_fingerprint_t *fpr);
+const pgp_key_t *rnp_key_store_get_key_by_fpr(const rnp_key_store_t *,
```

Comment:
It is `fpr` in this place and `fp` in others.
Can we rename it to have consistency throughout?

Reply:
@rrrooommmaaa Agree, I just left it as it was previously since it wasn't goal of this PR (and this function is just internal one). At some point we should make all internal naming consistent, i.e. keyid instead of key_id, and so on. Will file a separate issue regarding it.